Web/stockdogweb/src/components/Trade/Trade.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import './Trade.css';
+import Button from '../Button/Button';
+
+class Trade extends Component {
+   render() {
+      return (
+         <div className='Trade'>
+            <div className='trade-elements'>
+               <div className='trade-element'>
+                  <h1>13</h1>
```

Comment:
Are these numbers going to come from props?

Reply:
Yes let me change it